### ISSUE TITLE
Does MOODS take into account multiple testing correction?

### ISSUE THREAD

**Issue by doczmp** [jhkorhonen/MOODS#22]
Hello,

I was just wondering if MOODS uses some kind of multiple testing correction with regards to it p-values like FIMO? I could not find anything in the MOODS paper referring to it. 

Thanks

**Comment by jhkorhonen** [comment]
I guess you mean computing q-values basically? The answer is no; p-value thresholds are simply computed from a background model without further statistical testing.

(Overall, MOODS has been mostly focused on making the scanning part as fast as possible, and mostly assumes that the outputs are then post-processed according to the particular needs of the application at hand.)

**Comment by doczmp** [comment]
Yep, I was referring to q-values. Thanks a lot for the response.
